cmd/nerdctl/container_commit_test.go

```diff
@@ -53,3 +56,56 @@ func TestCommit(t *testing.T) {
 		base.Cmd("rmi", testImage).Run()
 	}
 }
+
+func kubeCtlHelper(base *testutil.Base, args ...string) *testutil.Cmd {
```

Comment:
nit:
```suggestion
func kubectlHelper(base *testutil.Base, args ...string) *testutil.Cmd {
```

This should be also moved to testutil

Reply:
Moved to testutil, renamed to `KubectlHelper`.